clients/src/main/resources/common/message/DescribeQuorumResponse.json

```diff
@@ -38,7 +38,9 @@
           "about": "The latest known leader epoch"},
         { "name": "HighWatermark", "type": "int64", "versions": "0+"},
```

Comment:
Does this return `-1` if the high watermark offset is not known? If so, how about adding an about that explain this.